src/datagen/java/blusunrize/immersiveengineering/data/IEBlockTags.java

```diff
@@ -95,9 +95,17 @@ protected void addTags()
 				{
 					assert tags.ore!=null;
 					tag(tags.ore).add(IEBlocks.Metals.ORES.get(metal).get());
+					tag(tags.ore).add(IEBlocks.Metals.DEEPSLATE_ORES.get(metal).get());
 					tag(Tags.Blocks.ORES).addTag(tags.ore);
+					assert tags.rawBlock!=null;
+					tag(tags.rawBlock).add(IEBlocks.Metals.RAW_ORES.get(metal).get());
 				}
 			}
+			else //TODO Forge#789, remove if/when we get tags for raw blocks
+			{
+				assert tags.rawBlock!=null;
```

Comment:
Can you replace the `assert`s with `Precondition.checkState` or in this case `Precondition.checkNonNull`? I don't think `assert`s are enabled by default, even in dev.

Reply:
What do you mean `assert`s are not enabled? Are you telling me the common Java keyword `assert` doesn't work in Minecraft?! O_o

Reply:
Less MC and more Java in general, at least according to https://stackoverflow.com/questions/29120928/why-is-assert-not-enabled-by-default-in-java

Reply:
As someone who uses Python primarily these days: What the fuck.
I mean, I get their approach, but wow that is so unlike what I'd expect...